shellcheck.properties

```diff
@@ -1,14 +1,20 @@
 category=Languages
 description=Enables scanning of shell scripts
 homepageUrl=https://github.com/sbaudoin/sonar-shellcheck
-archivedVersions=2.2.0,2.2.1,2.3.0
-publicVersions=2.4.0
+archivedVersions=2.2.0,2.2.1,2.3.0,2.4.0
+publicVersions=2.5.0
 
 defaults.mavenGroupId=com.github.sbaudoin
 defaults.mavenArtifactId=sonar-shellcheck-plugin
 
+2.5.0.description=Brings 54 new rules, fixes some minor issues
+2.5.0.sqVersions=[7.9,LATEST]
```

Comment:
all support for 7.9 is officially over. Please update the range start to the current LTS; 8.9